packages/@react-types/table/src/index.d.ts

```diff
@@ -10,70 +10,120 @@
  * governing permissions and limitations under the License.
  */
 
-import {AriaLabelingProps, AsyncLoadable, CollectionChildren, DOMProps, MultipleSelection, SectionProps, Sortable, StyleProps} from '@react-types/shared';
+import {AriaLabelingProps, AsyncLoadable, CollectionChildren, DOMProps, MultipleSelection, Sortable, StyleProps} from '@react-types/shared';
 import {GridCollection, GridNode} from '@react-types/grid';
 import {Key, ReactElement, ReactNode} from 'react';
 
 export interface TableProps<T> extends MultipleSelection, Sortable {
-  children: ReactElement<TableHeaderProps<T> | TableBodyProps<T> | SectionProps<T> | RowProps<T>>[],
+  // TODO: I think this shouldn't have Section and row? Also need to fix prop table rendering for this
```

Comment:
correct me if I'm mistaken here. Will need to figure out a good way to fix the prop table styling for situations like this

Reply:
One method would be to apply something like the following to the Type `td`:  "style={{'max-width': '200px', 'overflow-wrap': 'break-word'}}". Perhaps apply the other columns as well?

Reply:
![image](https://user-images.githubusercontent.com/9661127/115461519-5f61f900-a1de-11eb-8c58-2ce15e973e88.png)
Dependent on screen size, I'm inclined to make this a followup since it is an issue with the docs prop table in general I think

Reply:
oo, ok, that makes sense, hmmm... maybe followup is best